# Main params
alt: 2.0  # absolute alt, m
pub_freq: 150  # Hz

# Type of trajectory
traj_type: Circle  # Circle or Line

# Circular trajectory parameters
r: 3.4  # m
center_x: 0.0  # m
center_y: 0.5  # m
v_goals: [1.0, 2.0, 3.0, 4.0, 5.0]  # velocity magnitudes in increasing order, m/s
t_traj: 20.0  # time to follow each trajectory, s
circle_accel:  0.4  # max acceleration allowed, m/s^2. Should be small

# Line trajectory parameters
Ax: -4.25
Ay: -3.5
Bx: 4.5
By: 4.25
v_line: 3.0  # m/s
line_accel: 1.5
line_decel: 1.0  # should be smaller for safety

# Other params
vel_initpos: 0.4  # m/s horizontal speed towards initial point
vel_take: 0.3  # m/s of vertical speed
vel_land_fast: 0.35
vel_land_slow: 0.04
vel_yaw: 0.2

dist_thresh: 0.1  # m
yaw_thresh: 0.2  # rad

margin_takeoff_outside_bounds: 0.05  # m